test/test_base_service.py

```diff
@@ -154,6 +155,27 @@ def test_fail_http_config():
     with pytest.raises(TypeError):
         service.with_http_config(None)
 
+@responses.activate
+def test_cwd():
+    file_path = os.path.join(
+        os.path.dirname(__file__), '../resources/ibm-credentials.env')
+    # Try changing working directories to test getting creds from cwd
+    cwd = os.getcwd()
+    os.chdir(os.path.dirname(file_path))
+    iam_authenticator = get_authenticator_from_environment('ibm-watson')
+    os.chdir(cwd)
+    service = AnyServiceV1('2017-07-07', authenticator=iam_authenticator)
+    assert service.service_url == 'https://gateway.watsonplatform.net/test/api'
+    assert service.authenticator is not None
+
+    # Copy credentials file to cwd to test loading from current working directory
+    temp_env_path = os.getcwd() + '/ibm-credentials.env'
+    copyfile(file_path, temp_env_path)
+    iam_authenticator = get_authenticator_from_environment('ibm-watson')
+    os.remove(temp_env_path)
```

Comment:
This is the same as the first scenario, except in the case above you were "cd"'ing back to the original working directory (hence hiding the ibm-credentials.env file) and in this case you're actually deleting the file prior to constructing the service.   In both cases, the file will not be "seen" when the service is being constructed.

Reply:
I think what we need to test (primarily) is a simple scenario like this:
- change the CWD to the location where the `ibm-credentials.env` file exists
- load an authenticator using the factory and pass in "ibm_watson" for the service name; verify that it is the correct type of authenticator (based on the properties defined in `ibm-credentials.env`)
- construct an instance of the test service and call its `configure_service()` method (passing in "ibm_watson" for the service name; verify that the service url field of the service is set to the URL specified in `ibm-credentials.env`.

Reply:
Right, I mistakenly put my cleanup statements in the wrong place so the service was defaulting to the default URL. I fixed it as you said and it's working as expected now. Thansk!